packages/core/src/babelPlugins/babel-plugin-redwood-routes-auto-loader.ts

```diff
@@ -7,13 +7,15 @@ import { generateTypeDef, generateTypeDefIndex } from './generateTypes'
 
 interface PluginOptions {
   project: RWProject
+  prerendering: boolean
```

Comment:
🤔 Maybe we can call this `useStaticImports` and default to false, then it's more about what instead of why.

Reply:
Hmmm, yeah totally sensible. Remember the prerender path does two things:
1. Removes all existing imports for pages that the user added (this is so that we're consistent on how pages are loaded in Node.js environment)
2. uses static imports